Server/areas.h

```diff
@@ -22,6 +22,14 @@ typedef struct node {
  */
 int create_area(enum type type, char* name);
 
+/**
+ * @brief Verify if an area exists in the list
+ * @param name: name of the area to search
+ * @param head: head of the list
+ * @return int: 1 if the area exists, 0 otherwise
+ */
+int area_exists(char name[LENGTH_NAME_AREA], node_t * head);
```

Comment:
In functions parameters, I generally prefer to write `char* parameter` rather than `char parameter[...]`.
Sometimes it can create warnings at compile time, but if it doesn't this is good enough